src/thunderbots/software/ai/hl/stp/action/chip_action.cpp

```diff
@@ -93,10 +94,9 @@ void ChipAction::calculateNextIntent(IntentCoroutine::push_type& yield)
         bool robot_behind_ball = behind_ball_region.containsPoint(robot->position());
         // The point in the middle of the region behind the ball
         Point point_behind_ball =
-            chip_origin + behind_ball.norm(DIST_TO_FRONT_OF_ROBOT_METERS * 0.5 +
-                                           size_of_region_behind_ball / 2);
+            chip_origin + behind_ball.norm(size_of_region_behind_ball * 3 / 4);
 
-        // If we're not in position to chip, move into position
+        // If we're not in position to kick, move into position
```

Comment:
Similar comment to above, this should be `chip` still?

Reply:
fixed